src/components/oculus-touch-controls.js

```diff
@@ -59,14 +59,14 @@ var CONTROLLER_PROPERTIES = {
   'oculus-touch-v2': {
     left: {
       modelUrl: TOUCH_GEN2_CONTROLLER_MODEL_BASE_URL + 'gen2-left.gltf',
-      rayOrigin: {origin: {x: -0.01, y: 0, z: -0.02}, direction: {x: 0, y: -0.5, z: -1}},
-      modelPivotOffset: new THREE.Vector3(0, 0, 0),
+      rayOrigin: {origin: {x: 0, y: 0, z: 0}, direction: {x: 0, y: 0, z: -1}},
+      modelPivotOffset: new THREE.Vector3(0.01, 0.005, 0.03),
```

Comment:
Why does the model position need to be adjusted? I thought the problem was only with the ray position.

Reply:
It seems that the model is slightly out of position. I adjusted the offset so that the model position does not change when entering/leaving VR.
However it is only tested on my Oculus Quest, so I'll also consider to reset the change.

Reply:
Better to separate changes and put the model shift in a different PR if necessaryy. I still think that for the rayOrigin we should probably use InputSource values instead of hard coding numbers.

Reply:
Sorry for late, I reverted the change for model shift.
Also, it seems that XRInputSource.targetRaySpace has already been applied to the pose of controllers. https://github.com/aframevr/aframe/blob/master/src/components/tracked-controls-webxr.js#L115 So rayOrigin and direction may not be needed if XRInputSource is available.